src/drivers/imu/bmi055/BMI055.hpp

```diff
@@ -53,7 +53,8 @@
 class BMI055 : public device::SPI, public I2CSPIDriver<BMI055>
 {
 public:
-	BMI055(const char *name, const char *devname, I2CSPIBusOption bus_option, int bus, int type, uint32_t device,
+	BMI055(uint8_t devtype, const char *name, const char *devname, I2CSPIBusOption bus_option, int bus, int type,
```

Comment:
You now have `devtype` and `type`, which are both the same.

Reply:
Thanks, good eye. Fixed.